guides/common/modules/tuning-satellite-server-with-predefined-profiles.adoc

```diff
@@ -1,9 +1,9 @@
 [id='tuning-satellite-server-with-predefined-profiles']
 
-= Tuning Satellite Server with Predefined Profiles
+= Tuning {ProjectServer} with Predefined Profiles
 
-If your Satellite deployment includes more than 5000 hosts, you can use predefined tuning profiles to improve performance of Satellite Server.
-Choose one of the profiles depending on the number of hosts your Satellite manages and available hardware resources. The tuning profiles are available in the https://github.com/RedHatSatellite/satellite-support/tree/master/tuning-profiles[satellite-support GitHub repository].
+If your {Project} deployment includes more than 5000 hosts, you can use predefined tuning profiles to improve performance of {ProjectServer}.
+Choose one of the profiles depending on the number of hosts your {Project} manages and available hardware resources. The tuning profiles are available in the https://github.com/RedHat{Project}/satellite-support/tree/master/tuning-profiles[satellite-support GitHub repository].
```

Comment:
I imagine this link doesn't need the attribute.

Reply:
ffffff